src/service/resources-impl.js

```diff
@@ -1495,8 +1476,7 @@ export class Resource {
     /** @const {!AmpElement|undefined|null} */
     this.owner_ = undefined;
 
-    /** @const {number} */
-    this.priority_ = getElementPriority(element.tagName);
+    this.priority_ = element.getElementPriority();
```

Comment:
I don't understand how can I delegate to ElementProto -> BaseElement? I still need a `getPriority()` function inside the `Resource` class right? What call should I make here?